XCUITests/SettingsTest.swift

```diff
@@ -45,7 +45,7 @@ class SettingsTest: BaseTestCase {
             // Tap on "Default Browser App" and set the browser as a default (Safari is listed first)
             waitForExistence(iOS_Settings.tables.cells.element(boundBy: 1), timeout: 5)
             iOS_Settings.tables.cells.element(boundBy: 2).tap()
-            iOS_Settings.tables.staticTexts.element(boundBy: 1).tap()
+            iOS_Settings.tables.staticTexts["Firefox"].tap()
```

Comment:
That's not going to work... the name shown in Settings for the app is `Fennec (DeviceName)` :(

Reply:
Is this something we can add? something like` iOS_Settings.tables.staticTexts["Fennec (DeviceName)"]`?

Reply:
I have been trying to find that out but I don't see how to do that only within the test... other than adding the device name that Bitrise will show (`Fennec (vagrant)`) so that it works there I don't see an easy solution...